### ISSUE TITLE
row vs column

### ISSUE THREAD

**Issue by fabriziomoscon** [Kapiroska/crew#2]
Could you please explain to us your reasoning about using `CrewRow`? It would seem more appropriate to us naming something like Column, but maybe we missed something

**Comment by Kapiroska** [comment]
Yeah, it's my mistake. If I had had time for refactoring, I think I would have fixed it.
There are a lot of places where I should give more appropriate name
